__tests__/initial.migration.test.ts

```diff
@@ -14,13 +14,13 @@ import {
   IResolver,
   TAgent,
   VerifiableCredential,
-} from '../packages/core/src'
-import { DIDResolverPlugin } from '../packages/did-resolver/src'
-import { EthrDIDProvider } from '../packages/did-provider-ethr/src'
-import { WebDIDProvider } from '../packages/did-provider-web/src'
-import { getDidKeyResolver, KeyDIDProvider } from '../packages/did-provider-key/src'
-import { DIDComm, IDIDComm } from '../packages/did-comm/src'
-import { KeyManagementSystem, SecretBox } from '../packages/kms-local/src'
```

Comment:
these imports from `../packages/*/src` ensure that the tests use the source files directly instead of having to rebuild after every change.

Does it not work without this change?